### DISCUSSION TITLE
Farm Plot Directory / recursive

### DISCUSSION THREAD

**Discussion by josea** [Chia-Network/chia-blockchain#10592] [Ideas]
As discussed in  [reddit](https://www.reddit.com/r/chia/comments/t8shl9/where_to_post_suggestions_to_chia_plot_folders/) please add option so it can look into the subfolders of a given folder for existent plots. 

It would make life easier for systems with +100 physical drives.

**Comment by antonmakarenko** [comment]
I have a related suggestion: https://github.com/Chia-Network/chia-blockchain/discussions/11018
